flink-libraries/flink-state-processing-api/src/test/java/org/apache/flink/state/api/output/SnapshotUtilsTest.java

```diff
@@ -52,6 +53,7 @@
 
     @Test
     public void testSnapshotUtilsLifecycle() throws Exception {
+        ACTUAL_ORDER_TRACKING.clear();
```

Comment:
How about introduce a `@Before` method to clear the `ACTUAL_ORDER_TRACKING`?

Reply:
It can make code reuse.